Posts/2021/02/Get-Services, PowerShell 7 VS Windows PowerShell 5.1

```diff
@@ -0,0 +1,150 @@
+---
+post_title: Get-Services, PowerShell 7 VS Windows PowerShell 5.1
+username: farisnt@gmail.com
+Catagories: PowerShell, Windows Services
+Summary: Show how to get services information using both Windows PowerShell 5.1 and PowerShell 7
+---
+
+# Get-Services, PowerShell 7 VS Windows PowerShell 5.1
+
+**Q**: How can I get the Username, and StartType for a Windows Service?
+
+**A**: Quick answer is PowerShell 7.
+
+Microsoft is doing a great job on Powershell on each version they release,  to answer this question a simple command called `Get-Service` will be used, but there is a big update that makes getting the required information way much easier with PowerShell 7. I will show the result of this command using both **PowerShell 7** and **Windows PowerShell 5.1**.
+
+Let's start by typing the simple command **`Get-Service Workstation`.** This command will get very basic details for a service called *Workstation.* The result will be the same for both Powershell 7 and Windows Powershell 5.1
```

Comment:
This cmdlet returns basic details for a service called *Workstation*.
The result is the same for PowerShell 8 and Windows PowerShell 5.1